### PR TITLE
Add a schema_migrations_ran

### PR DESCRIPTION
**Pull request by bdunne**
Add a table to track which migrations have run on which regions

https://bugzilla.redhat.com/show_bug.cgi?id=1668800

### CODE REVIEW

File: db/migrate/20171031010000_add_schema_migrations_ran.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+class AddSchemaMigrationsRan < ActiveRecord::Migration[5.0]
+  def up
+    create_table :schema_migrations_ran do |t|
+      t.string :version
+      t.datetime :created_at, :null => false
```
Comment (carbonin) [inline db/migrate/20171031010000_add_schema_migrations_ran.rb]: I thought we were not going to add this here. Why the change now?

**Comment by simaishi** [comment]
Hammer backport details:

```
$ git log -1
commit 4f37c61dbad2323bb89c2233bd6adbb1f3f7ba1f
Author: Nick Carboni <ncarboni@redhat.com>
Date:   Tue Jan 29 13:01:25 2019 -0500

    Merge pull request #330 from bdunne/schema_migrations_ran
    
    Add a schema_migrations_ran
    
    (cherry picked from commit 3b098574432f424058f485f11b7c72479dcf6a4c)
    
    https://bugzilla.redhat.com/show_bug.cgi?id=1668800
```
